docs/en/latest/apisix-variable.md

```diff
@@ -39,5 +39,6 @@ List in alphabetical order:
 | route_name       | core    | name of `route`        |   |
 | service_id       | core    | id of `service`        |   |
 | service_name     | core    | name of `service`      |   |
+| redis_cmd_line   | Redis   | the content of Redis command |   |
```

Comment:
Better shorten name `redis_cmd` or `redis_command`?

`redis_cmd_line` seems it'll return multi commands compact into one line. It just return one total command content, not just a redis command like `hmset`, right?

Reply:
yes, what it really means is command line.